src/Bridge/Doctrine/Filter/AbstractFilter.php

```diff
@@ -0,0 +1,403 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Doctrine\Filter;
```

Comment:
I was surprised because there is no diff. But yeah, we moved this in `ApiPlatform\Core\Bridge\Doctrine\Orm\Filter` since this is why there are no diffs :p. 
My bad!